packages/create-instance/add-slots.js

```diff
@@ -1,60 +1,39 @@
 // @flow
 
 import { compileToFunctions } from 'vue-template-compiler'
-import { throwError } from 'shared/util'
 import { validateSlots } from './validate-slots'
+import { toArray } from 'shared/util'
 
-function addSlotToVm (vm: Component, slotName: string, slotValue: Component | string | Array<Component> | Array<string>): void {
-  let elem
-  if (typeof slotValue === 'string') {
-    if (!compileToFunctions) {
-      throwError('vueTemplateCompiler is undefined, you must pass components explicitly if vue-template-compiler is undefined')
-    }
-    if (typeof window === 'undefined') {
-      throwError('the slots string option does not support strings in server-test-uitls.')
-    }
-    if (window.navigator.userAgent.match(/PhantomJS/i)) {
-      throwError('the slots option does not support strings in PhantomJS. Please use Puppeteer, or pass a component.')
-    }
-    const domParser = new window.DOMParser()
-    const _document = domParser.parseFromString(slotValue, 'text/html')
-    const _slotValue = slotValue.trim()
-    if (_slotValue[0] === '<' && _slotValue[_slotValue.length - 1] === '>' && _document.body.childElementCount === 1) {
-      elem = vm.$createElement(compileToFunctions(slotValue))
-    } else {
-      const compiledResult = compileToFunctions(`<div>${slotValue}{{ }}</div>`)
-      const _staticRenderFns = vm._renderProxy.$options.staticRenderFns
```

Comment:
I removed this 'replacing' of `staticRenderFns ` and didn't experienced unit-test failures - is it ok to remove, or I miss something ?

Reply:
I don't think it's OK to remove, but we should add some tests to cover it. Can you write a test to check that a static slot is added correctly.

Reply:
Thanks @eddyerburgh for information - I'll add appropriate tests for that

Reply:
I moved back `staticRenderFns` related code in pull request. To be honest I don't know what is static slot - I'm not yet familiar with everything related with Vue.